samza-core/src/main/java/org/apache/samza/config/JavaSystemConfig.java

```diff
@@ -60,4 +66,33 @@ public String getSystemFactory(String name) {
     }
     return systemNames;
   }
+
+  /**
+   * @return map of system name to {@link SystemAdmin}
+   */
+  public Map<String, SystemAdmin> getSystemAdmins() {
```

Comment:
Should we update Scala SystemConfig for parity as well? Not sure what the difference b/w the two is.